src/digraph.rs

```diff
@@ -1503,6 +1553,25 @@ impl PyDiGraph {
                 .collect(),
         }
     }
+
+    /// Return the list of edge indices incent to a provided node
+    ///
+    /// :param int node: The node index to get incident edges from. If
+    ///     this node index is not present in the graph this method will
+    ///     return an empty list and not error.
+    ///
+    /// :returns: A list of the edge indices incident to a node in the graph
+    /// :rtype: EdgeIndices
+    pub fn incident_edges(&self, node: usize) -> EdgeIndices {
```

Comment:
```suggestion
    #[pyo3(text_signature = "(self, node, /)")]
    pub fn incident_edges(&self, node: usize) -> EdgeIndices {
```

It should also be mentioned in the docs that this function returns only outgoing incident edges.

Reply:
In https://github.com/Qiskit/retworkx/pull/538/commits/e737ca7531995665d1e16f589a6720b52e1e8700 I've updated the digraph method to have an `all_edges` option to return both outgoing and incoming edges and updated the documentation to reflect by default it's only outgoing but if you want incoming you can use that flag.